packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js

```diff
@@ -1,26 +1,29 @@
+import EventEmitter from 'events';
+
 import {isEqual} from 'lodash';
 
 import Events from '../events';
-import {EVENTS, LOCUSEVENT} from '../constants';
+import {EVENTS, LOCUSEVENT, USER, ONE_ON_ONE} from '../constants';
 import InfoUtils from '../locus-info/infoUtils';
 import SelfUtils from '../locus-info/selfUtils';
 import HostUtils from '../locus-info/hostUtils';
 import MediaSharesUtils from '../locus-info/mediaSharesUtils';
 import Parser from '../locus-info/parser';
 
+
 /**
  * @export
  * @class LocusInfo
  */
-export default class LocusInfo {
+export default class LocusInfo extends EventEmitter {
   constructor(callback, locus = {}, spark) {
+    super();
```

Comment:
@andrewholsted so in this case we have an event emitter that is outside of the global event emitter, do you have any concerns with this? it enables us to listen to events from locus-info objects and separate from global events, which is definitely beneficial, but just curious if you know of anything?

Reply:
id like to hear from andrew on this one still

Reply:
I have concerns about this

Reply:
Only an consumer subscribing to events from LocusInfo will get called. I don't see why we could use the other EventEmitter that already exists?

Reply:
In this case locus-info doesnt send event to consumer . Meetings--> <-- locus-Info the events sent by locus-info is only cared by meeting object. So didnt feel the necessary for global Event emitter.  It creates more confusion for parent child communication

Reply:
We already have an `/events` which contains an event emitter

Reply:
That one is used for communication between files and doesn't emit to the client/end user

Reply:
We discussed and decided this is ok because the parent listener only cares about scoped events from it's direct child. Not all events.

Reply:
> We discussed and decided this is ok because the parent listener only cares about scoped events from it's direct child. Not all events.

@arun3528 please add this as a comment or description to this particular emitter. I think it's confusing to have the separate ones unless we're explicit about why. 

A different name may help: `ScopedEmitter`? `ChildEmitter`?